packages/babel-node/README.md

```diff
@@ -0,0 +1,19 @@
+# babel-node
+
+> Babel node
+
+babel comes with a second CLI which works exactly the same as Node.js’s CLI, only it will compile ES6 code before running it.
+
+## Install
+
+```sh
+npm install --save-dev babel-node
+```
+
+## Usage
+
+```sh
+babel-node [options] [ -e script | script.js ] [arguments]
+```
+
+For more in depth documentation see: http://babeljs.io/docs/usage/cli/#babel-node
```

Comment:
Don't think we need this line here (otherwise it'll get output on the website :D)